### PR TITLE
Added Smooth Navigation transition in sidebar menu

### LINKED ISSUE #1614: [Brand] Add smooth navigation transition to sidebar menu
**Current Behavior**
On the https://layer5.io/brand page, when a user clicks on the links in the sidebar navigator (see screenshot), the transition to the respective section is not smooth.

Screen Shot 2021-02-25 at 9 20 09 PM

**Desired Behavior**
Upon click, the page should smoothly scroll (transition) to the desired section and not do so abruptly.

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs).

### PR DESCRIPTION
**Pull request by chinmaym07**
Signed-off-by: chinmaym07 <b418020@iiit-bh.ac.in>

**Description**

This PR fixes #1614 

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: src/sections/Company/Brand/index.js [hunk 1]
```diff
@@ -32,11 +32,40 @@ const getDimensions = ele => {
   };
 };
 
-const scrollTo = ele => {
-  ele.scrollIntoView({
-    behavior: "smooth",
-    block: "start",
-  });
+const scrollTo = (element,duration=2000) => {
+  var e = document.documentElement;
```
Comment (Nikhil-Ladha) [inline src/sections/Company/Brand/index.js:36]: Use `let` instead

File: src/sections/Company/Brand/index.js [hunk 2]
```diff
@@ -32,11 +32,40 @@ const getDimensions = ele => {
...
+  if(e.scrollTop===0){
+    var t = e.scrollTop;
```
Comment (Nikhil-Ladha) [inline src/sections/Company/Brand/index.js:38]: Same as above

**Review by Nikhil-Ladha — COMMENTED** [review]
Please increase code readability....use proper indentation wherever needed.

**Review by leecalcote — APPROVED** [review]

File: src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx [hunk 3]
```diff
@@ -15,7 +15,7 @@ import layer5_hashicorp_partnership from "./Layer5-HashiCorp-Service-Mesh-Partne
 import layer5_image_hub from "./layer5-image-hub-on-hashicorp-consul.png";
 
 <NewsWrapper>
-
+<link rel="canonical" href="/company/news/layer5-and-hashicorp-launch-service-mesh-partnership" />
```
Comment (Nikhil-Ladha) [inline src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx:18]: Why this change?

Comment (Nikhil-Ladha) [reply src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx:18]: Ok, I think Lee added this....merging 👍🏼

**Review by Nikhil-Ladha — APPROVED** [review]

File: src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx (hunk 3 above)
Comment (leecalcote) [reply src/collections/news/2020-05-28-layer5-and-hasicorp-launch-service-mesh-partnership/index.mdx:18]: Oops, yes, I was checking out @chinmaym07's update and fixed another item while in there, not realizing I was pushing to Chinmay's branch.
